crates/swc_bundler/src/bundler/chunk/cjs.rs

```diff
@@ -266,20 +266,24 @@ where
                 // TODO
                 for spec in i.specifiers.clone() {
                     match spec {
-                        ImportSpecifier::Named(s) => {
-                            if let Some(imported) = s.imported {
+                        ImportSpecifier::Named(s) => match s.imported {
+                            Some(ModuleExportName::Ident(imported)) => {
                                 props.push(ObjectPatProp::KeyValue(KeyValuePatProp {
                                     key: imported.into(),
                                     value: Box::new(s.local.into()),
                                 }));
-                            } else {
+                            }
+                            Some(ModuleExportName::Str(..)) => {
+                                unimplemented!("module string names unimplemented")
+                            }
```

Comment:
I used some time to think about this, but I concluded that panicking is fine for the bundler at the moment.

Reply:
I'll fix this in near future.